src/fund.rs

```diff
@@ -0,0 +1,63 @@
+use ethereum_types::{Address, H256, U256};
+use ethcore::{self, state::{State, CleanupMode}};
+use oasis_ethwasi_runtime_common::parity::NullBackend;
+extern crate chrono;
+use chrono::{DateTime, NaiveDateTime, Utc, Datelike};
+extern crate date_time;
+use date_time::month_tuple::MonthTuple;
+use std::str::FromStr;
+use std::convert::TryInto;
+use byteorder::{BigEndian, ReadBytesExt};
+
+fn timestamp_to_mt(timestamp:i64) -> MonthTuple{
```

Comment:
Should we run a fmt tool for checking the coding style?

Reply:
Yes, i will accomplish it later.